### ISSUE TITLE
scale values of the teacher network

### ISSUE THREAD

**Issue by samo313** [i13abe/Triplet-Loss-for-Knowledge-Distillation#1]
Hi, I use your pre-train cnn_alex.pkl as a teacher in my code but It gives me a high scale value range in the last encoding vector. I have a question about this model Is it the base Alexnet model or you trained it by yourself?

**Comment by i13abe** [comment]
Hi

Thank you for e-mail.
It is trained model by myself.
I implemented Alexnet-like model in network.py and I used it.
If you want to use your own teacher model, you can change Net_teacher() at
line 82 in example.py to your own model.
Also please change model_t.load_state_dict(torch.load(''cnn_alex.pkl'')) to
own pkl file.

I hope your best.

Motoshi Abe


2021年4月24日(土) 午後11:32 samo313 ***@***.***>:

**Comment by i13abe** [comment]
It is trained model by myself.
I implemented Alexnet-like model in network.py and I used it.
If you want to use your own teacher model, you can change Net_teacher() at line 82 in example.py to your own model.
Also please change model_t.load_state_dict(torch.load(''cnn_alex.pkl'')) to own pkl file.

**Comment by samo313** [comment]
> It is trained model by myself.
> I implemented Alexnet-like model in network.py and I used it.
> If you want to use your own teacher model, you can change Net_teacher() at line 82 in example.py to your own model.
> Also please change model_t.load_state_dict(torch.load(''cnn_alex.pkl'')) to own pkl file.

Thank you so much.
